crates/hyperqueue/src/client/commands/server.rs

```diff
@@ -70,17 +106,40 @@ pub async fn command_server(gsettings: &GlobalSettings, opts: ServerOpts) -> any
         ServerCommand::Start(opts) => start_server(gsettings, opts).await,
         ServerCommand::Stop(opts) => stop_server(gsettings, opts).await,
         ServerCommand::Info(opts) => command_server_info(gsettings, opts).await,
+        ServerCommand::GenerateAccess(opts) => command_server_generate_access(gsettings, opts),
     }
 }
 
 async fn start_server(gsettings: &GlobalSettings, opts: ServerStartOpts) -> anyhow::Result<()> {
+    let access_file: Option<FullAccessRecord> = if let Some(path) = opts.access_file {
+        let file = File::open(path)?;
```

Comment:
This code could live somewhere "closer" to where the access record is actually generated, so that we make sure that the format stays the same (if we e.g. switch from JSON to something else). It should be near the `store_access_record` function.